ci/docker/linux-apt-lint.dockerfile

```diff
@@ -40,7 +40,7 @@ RUN apt-get update && \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-ARG r=4.1
+ARG r=4.2
 RUN wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | \
         tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc && \
     # NOTE: R 3.5 and 3.6 are available in the repos with -cran35 suffix
```

Comment:
I can't make a suggestion spanning this whole block, but delete this comment to say that only R >= 4.0 is available in this repo. 

Maintaining this hack so that we could technically support R 3.5 or 3.6, [but only on Ubuntu 18.04 or earlier](https://cloud.r-project.org/bin/linux/ubuntu/olderreleasesREADME.html), is more harm than help, as you experienced in this PR. We don't have any jobs that do that. When we test older R versions, we either use `r-lib/actions/setup-r` or Docker images.

Reply:
Done